frontend/src/container/Account.jsx

```diff
@@ -77,7 +77,10 @@ class Account extends React.Component {
 
   customEventReducer = ({ detail: { type, data } }) => {
     switch (type) {
-      case CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE: this.buildBreadcrumbs(); break
+      case CUSTOM_EVENT.ALL_APP_CHANGE_LANGUAGE:
+        this.buildBreadcrumbs()
+        this.setHeadTitle(this.props.t('My Account'))
```

Comment:
this will break once the PR about merge generic code of apps is merged.
We'll see what to do